doc/source/whatsnew/v2.0.0.rst

```diff
@@ -161,6 +161,7 @@ Alternatively, copy on write can be enabled locally through:
 
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
+- Added support for ``dt`` accessor methods when using :class:`ArrowDtype` with a datetime-like type (:issue:`50954`)
```

Comment:
might be worth clarifying what datetime-like means here since it doesn't match the non-pyarrow usage

Reply:
Yeah makes sense. Clarified that this is for `pyarrow.timestamp` types